src/Transactions.css

```diff
@@ -0,0 +1,27 @@
+li {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    max-width: 600px;
+    margin: auto;
+    background: beige;
+    
+}
+
+.logo {
+    padding: 5px;
+}
+
+.description {
```

Comment:
generell die frage: sollen die klassen hier so generisch heißen? wenn sie nicht zu spezialisiert sind sondern uahc in anderen zusammenhängen genutzt werden können ok. aber wenn sie zb nur in listen anwendung finden könnte man es etwas genauer machen zb `list-item-description`

Reply:
Das CSS ist eigentlich noch placeholder... ich hab eigentlich vor styled-components einzusetzen aber wollte das iterativ angehen 🙄